### PR TITLE
[App Service] fix#-13035- Added validation for az webapp config access-restriction …

### PR DESCRIPTION
**Pull request by Kotasudhakarreddy**
**Description**  
#13035 
We cannot add duplicate IPs from the portal since it checks if a value exists however CLI allows this

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
App Service

**Comment by Kotasudhakarreddy** [comment]
These tests are failing because of `network` commands. I am unable to rerecord it in local as well. PFB for error details. please let me know who can help with the failing network tests.
image

**Comment by MyronFanQiu** [comment]
> These tests are failing because of `network` commands. I am unable to rerecord it in local as well. PFB for error details. please let me know who can help with the failing network tests.
> image

Please run `azdev setup` again.

**Comment by yungezz** [comment]
hi @qwordy could you pls help on reviewing? thanks

File: src/azure-cli/azure/cli/command_modules/appservice/_validators.py [hunk 1]
```diff
@@ -184,3 +190,15 @@ def validate_ip_address(namespace):
...
+
+
+def _validate_ip_address_existance(cmd, namespace):
+    resource_group_name = namespace.resource_group_name
+    name = namespace.name
+    slot = namespace.slot
+    from ._appservice_utils import _generic_site_operation
+    configs = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'get_configuration', slot)
+    access_rules = configs.ip_security_restrictions
+    is_exists = [(lambda x: x.ip_address == namespace.ip_address)(x) for x in access_rules]
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/_validators.py:202]: Is it necessary to use lambda here?

File: src/azure-cli/azure/cli/command_modules/appservice/_validators.py [hunk 2]
```diff
@@ -184,3 +190,15 @@ def validate_ip_address(namespace):
...
+    if True in is_exists:
+        raise CLIError('Rule is already exists with the specified criteria')
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/appservice/_validators.py:204]: Grammar error

File: src/azure-cli/azure/cli/command_modules/appservice/_validators.py (hunk 1 above)
Comment (Kotasudhakarreddy) [reply src/azure-cli/azure/cli/command_modules/appservice/_validators.py:202]: Yes with out lambda it is not yielding the output.

**Comment by Kotasudhakarreddy** [comment]
> > These tests are failing because of `network` commands. I am unable to rerecord it in local as well. PFB for error details. please let me know who can help with the failing network tests.
> > image
> 
> Please run `azdev setup` again.

did `azdev setup` again but no luck.

File: src/azure-cli/azure/cli/command_modules/appservice/_validators.py (hunk 2 above)
Comment (Kotasudhakarreddy) [reply src/azure-cli/azure/cli/command_modules/appservice/_validators.py]: Done the changes

**Pull request merged**